Lib/test/test_pyexpat.py

```diff
@@ -755,5 +755,83 @@ def resolve_entity(context, base, system_id, public_id):
         self.assertEqual(handler_call_args, [("bar", "baz")])
 
 
+class ReparseDeferralTest(unittest.TestCase):
+    def test_getter_initial_value(self):
+        if expat.version_info >= (2, 6, 0):
+            expected = True
+        else:
+            expected = False
+
+        parser = expat.ParserCreate()
+        actual = parser.GetReparseDeferralEnabled()
+
+        self.assertEqual(actual, expected)
+
+    def test_getter_setter_round_trip(self):
+        parser = expat.ParserCreate()
+        was_enabled = parser.GetReparseDeferralEnabled()
+
+        # We'll flip the value back and forth, and we expect ..
+        if expat.version_info >= (2, 6, 0):
+            # that flipping worked both ways for >=2.6.0
+            expected_1 = not was_enabled
+            expected_2 = was_enabled
+        else:
+            # that flipping did not have any effect for <2.6.0
+            expected_1 = False
+            expected_2 = False
+
+        parser.SetReparseDeferralEnabled(expected_1)
+        actual_1 = parser.GetReparseDeferralEnabled()
+
+        self.assertEqual(actual_1, expected_1)
+
+        parser.SetReparseDeferralEnabled(expected_2)
+        actual_2 = parser.GetReparseDeferralEnabled()
+
+        self.assertEqual(actual_2, expected_2)
+
+    def test_reparse_deferral_enabled(self):
+        if expat.version_info < (2, 6, 0):
+            return
```

Comment:
Use the `unittest.skipIf()` decorator or `self.skipTest()` or `raise unittest.SkipTest()`.